### DISCUSSION TITLE
GIthub Pages in private repo link not visible

### DISCUSSION THREAD

**Discussion by arawat-vartopia** [community/community#51305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,

I need your help with GitHub pages. I have created a public repository in which I have set up my angular project using GitHub pages which has a url. The problem i am facing when i make this repository to private its stop working.

My expectation is I want a private repo in which I can give the selected user access to view link.

Please help it out.

**Comment by IOLOII** [comment]
![image](https://user-images.githubusercontent.com/34856171/230585387-bb63877d-9524-4ab9-b705-c382d4a41481.png)

To be a GitHub Enterprise account (Need $).
